parm/config/gfs/config.resources

```diff
@@ -236,22 +236,13 @@ elif [[ "${step}" = "landanlinit" || "${step}" = "landanlrun"  || "${step}" = "l
    export layout_x
    export layout_y
 
-   if [[ "${step}" = "landanlinit" || "${step}" = "landanlfinal" ]]; then
-       declare -x "wtime_${step}"="00:10:00"
-       declare -x "npe_${step}"=1
-       declare -x "nth_${step}"=1
-       temp_stepname="nth_${step}"
-       declare -x "npe_node_${step}"="$(echo "${npe_node_max} / ${!temp_stepname}" | bc)"
-       declare -x "memory_${step}"="3072M"
-   elif [[ "${step}" = "landanlrun" ]]; then
-       export wtime_landanlrun="00:30:00"
-       npe_landanlrun=$(echo "${layout_x} * ${layout_y} * 6" | bc)
-       export npe_landanlrun
-       export nth_landanlrun=1
-       npe_node_landanlrun=$(echo "${npe_node_max} / ${nth_landanlrun}" | bc)
-       export npe_node_landanlrun
-       export is_exclusive=True
-   fi
+   export wtime_landanl="00:15:00"
+   npe_landanl=$(echo "${layout_x} * ${layout_y} * 6" | bc)
+   export npe_landanl
+   export nth_landanl=1
+   npe_node_landanl=$(echo "${npe_node_max} / ${nth_landanl}" | bc)
+   export npe_node_landanl
+   export is_exclusive=True
```

Comment:
@jiaruidong2017 @CoryMartin-NOAA Do we really need exclusive node?

Reply:
I have no idea on this. @CoryMartin-NOAA would you please answer this?

Reply:
probably not but I don't know how much memory it uses

Reply:
I will remove the `is_exclusive=True`.  If you need more memory, please determine how much rather than reserving the full node for a 6 processor job.